subsys/bluetooth/mesh/shell.c

```diff
@@ -898,6 +898,44 @@ static int cmd_gatt_proxy(const struct shell *shell, size_t argc, char *argv[])
 	return 0;
 }
 
+static int cmd_network_transmit(const struct shell *shell,
+		size_t argc, char *argv[])
+{
+	uint8_t transmit;
+	int err;
+
+	if (argc < 2) {
+		err = bt_mesh_cfg_network_transmit_get(net.net_idx,
+				net.dst, &transmit);
+	} else {
+		if (argc != 3) {
+			return -EINVAL;
```

Comment:
Please add a `shell_error` here, informing about the right number of parameters.